djstripe/forms.py

```diff
@@ -7,9 +7,11 @@
 
 """
 
+from .models import Plan
 from django import forms
 
-from .settings import PLAN_CHOICES
+
+PLAN_CHOICES = [(plan.stripe_id, plan.name) for plan in Plan.objects.all()]
```

Comment:
This might be needed in a few different places. I'd suggest converting it into a callable and moving it into djstripe/settings.py